### ISSUE TITLE
Add web interface 

### ISSUE THREAD

**Issue by manpreet9112** [manpreet9112/JavascriptGraphs#2]

**Comment by ghost** [comment]
So you wanna have web page that is styled with more then just a graph on white background ?

**Comment by manpreet9112** [comment]
No, I want to make it dynamic means whenever user change formula, graph should be change automatically.
